exercises/perfect-numbers/tests/perfect-numbers.rs

```diff
@@ -0,0 +1,35 @@
+extern crate perfect_numbers;
+
+use perfect_numbers::{Classification, classify};
+
+#[test]
+fn basic() {
+    assert_eq!(classify(0).unwrap_err(), "Number must be positive");
+}
+
+#[test]
+#[ignore]
+fn test_all() {
+    struct TestClassification {
+        num: u64,
+        result: perfect_numbers::Classification
+    }
+    let test_table: Vec<TestClassification> = vec![
+        TestClassification { num: 6, result: Classification::Perfect },
+        TestClassification { num: 28, result: Classification::Perfect },
+        TestClassification { num: 33550336, result: Classification::Perfect },
+        TestClassification { num: 33550336, result: Classification::Perfect },
```

Comment:
can you confirm whether I am seeing things, or whether this line appears twice in this file? lines 20 and 21 look the same to me

Reply:
Line 21 removed